### PR TITLE
Templates2

### PR DESCRIPTION
**Pull request by MFernirjons**
Read changes carefull especially you viggo

### CODE REVIEW

File: media/logos/Adams_Group.png
Comment (evaxrut) [inline media/logos/Adams_Group.png:1]: taka út öll logos

File: applicant/views.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def companies(request):
 def company_detail(request, cid):
     # return HttpResponse(f"This is the detail view for company {cid}.")
     # return render(request, 'applicant/company_detail.html', {cid: "dataset"})
-    company = Company.objects.get(id=cid)
+    company = Company.objects.filter(id=cid)
```
Comment (ViggoRu21) [inline applicant/views.py:77]: bad

Comment (ViggoRu21) [reply applicant/views.py:77]: filter -> get

**Review by ViggoRu21 — APPROVED** [review]
swag jesus

**Review by evaxrut — APPROVED** [review]
scared....

**Pull request merged**